### PR TITLE
Feature hero section

### PR DESCRIPTION
**Pull request by keptup**
Created Hero section, added temporary background image, chose website fonts, added initial animation for the background and centre container.
@spukas, please review the changes.

### CODE REVIEW

File: src/components/Hero/Hero 2.css [hunk 1]
```diff
@@ -0,0 +1,79 @@
+.hero-section {
```
Comment (spukas) [inline src/components/Hero/Hero 2.css]: If you are not using this file, please remove from the pull request.

File: src/components/Hero/Hero.css [hunk 2]
```diff
@@ -0,0 +1,98 @@
...
+    justify-content: center;
+    align-items: center;
+    height: 100vh;
+    width: 100%;
+    background-color: black;
+    background-image: url("../../images/temporary-image.jpg");
+    background-repeat: no-repeat;
+    background-position: center center;
+    background-size: auto 100%;
+    -webkit-animation: zoom 1s;
```
Comment (spukas) [inline src/components/Hero/Hero.css]: react-create-app adds vendor prefixes automatically so you do not need to add them manually.

File: src/components/Hero/Hero.css [hunk 3]
```diff
@@ -0,0 +1,98 @@
...
+}
+
+.hero-section__overlay {
+    position: absolute;
+    height: 100%;
+    width: 100%;
+    -webkit-animation: fadeout 1s;
+       -moz-animation: fadeout 1s;
+         -o-animation: fadeout 1s;
+            animation: fadeout 1s;
```
Comment (spukas) [inline src/components/Hero/Hero.css]: css properties should be aligned in one line from the left. This file is not linted

File: src/components/Hero/Hero.css [hunk 4]
```diff
@@ -0,0 +1,98 @@
...
+    @-moz-keyframes fadeout {
+          0% {background-color: black; opacity: 1;}
+        100% {opacity: 0;}
+    }
+    @-o-keyframes fadeout {
+          0% {background-color: black; opacity: 1;}
+        100% {opacity: 0;}
+    }
+    @keyframes fadeout {
+          0% {background-color: black; opacity: 1;}
```
Comment (spukas) [inline src/components/Hero/Hero.css]: after the curly braces are opened, the properties should start in the new line.

File: src/components/Hero/Hero.js [hunk 5]
```diff
@@ -1,8 +1,17 @@
...
+
+import './Hero.css';
 
 function Hero() {
     return (
-        <h2>This is a Hero section</h2>
+        <div className="hero-section">
+            <div className="hero-section__overlay"></div>
+            <div className="hero-section__center-container">
+                <div className="hero-section__logo-container"><img src={vataipvatLogo} alt="vataipvat-logo"></img></div>
```
Comment (spukas) [inline src/components/Hero/Hero.js]: Please move image tag to new line and use shorthand: ``

File: src/index.css [hunk 6]
```diff
@@ -5,9 +5,18 @@
...
 body {
-  font-family: -apple-system, BlinkMacSystemFont, "Segoe UI", "Roboto", "Oxygen",
+  font-family: "Helvetica Neue", "Open Sans", -apple-system, BlinkMacSystemFont, "Segoe UI", "Roboto", "Oxygen",
     "Ubuntu", "Cantarell", "Fira Sans", "Droid Sans", "Helvetica Neue",
     sans-serif;
   -webkit-font-smoothing: antialiased;
   -moz-osx-font-smoothing: grayscale;
+}
+
+/* :root {
```
Comment (spukas) [inline src/index.css]: Please remove commented and not used code.

File: src/App.css [hunk 7]
```diff
@@ -1,3 +1,4 @@
 .App {
-  text-align: initial;
+  display: flex;
+  flex-direction: column;
```
Comment (spukas) [inline src/App.css]: Why do you use flex property to position elements in column direction? Tags like `div` have display property of `block` and gets positioned to column direction by default.

File: src/components/Hero/Hero.css [hunk 8]
```diff
@@ -0,0 +1,98 @@
+.hero-section {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    height: 100vh;
```
Comment (spukas) [inline src/components/Hero/Hero.css]: Why are you using `vh` property here?

File: src/components/Hero/Hero.js (hunk 5 above)
Comment (spukas) [inline src/components/Hero/Hero.js]: `<div className="hero-section__logo-container">` is not needed and you are not styling it.
You can apply class name for `img` tag

File: src/components/Hero/Hero.css [hunk 9]
```diff
@@ -0,0 +1,98 @@
...
+        100% {opacity: 0;}
+    }
+    @keyframes fadeout {
+          0% {background-color: black; opacity: 1;}
+        100% {opacity: 0;}
+    }
+
+.hero-section__center-container {
+    display: flex;
+    flex-direction: column;
```
Comment (spukas) [inline src/components/Hero/Hero.css]: Why to use flex here? Image and h3 tags are positioned in new lines by default. You are abusing flex property too much :) Use it only when you really need it.

File: src/components/Hero/Hero.css [hunk 10]
```diff
@@ -0,0 +1,98 @@
...
+    align-items: flex-start;
+    margin: 0 6em;
```
Comment (spukas) [inline src/components/Hero/Hero.css]: Why do you use margins here? And why specifically 6em?

File: src/components/Hero/Hero.css (hunk 3 above)
Comment (keptup) [reply src/components/Hero/Hero.css]: Fixed

File: src/components/Hero/Hero.css (hunk 4 above)
Comment (keptup) [reply src/components/Hero/Hero.css]: Fixed

File: src/index.css (hunk 6 above)
Comment (keptup) [reply src/index.css]: Fixed

File: src/components/Hero/Hero.css (hunk 2 above)
Comment (keptup) [reply src/components/Hero/Hero.css]: Fixed.

File: src/components/Hero/Hero 2.css (hunk 1 above)
Comment (keptup) [reply src/components/Hero/Hero 2.css]: Fixed

File: src/components/Hero/Hero.js (hunk 5 above)
Comment (keptup) [reply src/components/Hero/Hero.js]: Fixed

File: src/App.css (hunk 7 above)
Comment (keptup) [reply src/App.css]: Fixed

File: src/components/Hero/Hero.css (hunk 8 above)
Comment (keptup) [reply src/components/Hero/Hero.css]: Percent property in this case does not allow to span the hero section to full height of the viewport.

File: src/components/Hero/Hero.js (hunk 5 above)
Comment (keptup) [reply src/components/Hero/Hero.js]: Fixed

File: src/components/Hero/Hero.css (hunk 9 above)
Comment (keptup) [reply src/components/Hero/Hero.css]: Fixed

File: src/components/Hero/Hero.css (hunk 10 above)
Comment (keptup) [reply src/components/Hero/Hero.css]: Fixed

**Pull request merged**